app/services/record_declaration.rb

```diff
@@ -156,10 +156,23 @@ def declaration_parameters
       declaration_type:,
       cpd_lead_provider:,
       delivery_partner:,
+      mentor_user_id: mentor&.id,
       user: participant_identity.user,
     }
   end
 
+  def mentor
```

Comment:
suggestion: could we name this `mentor_user` to avoid confusing it with a profile?

Reply:
I'll rename to `mentor_user_id` as it looks like I can drop the `.user` access and just grab `user_id` of the participant identity